Sprite/Sprite.js

```diff
@@ -254,7 +254,7 @@ const SpriteBase = kind({
 
 			for (let i = 0; i < sets; i++) {
 				const axisValue = (i * axis * -1);
-				const dimentionValue = ((steps - 0) * dimension * -1);
```

Comment:
here the calculation `- 0` seemed useless

Reply:
This might be use for type coercion. To ensure that we are dealing with a number. We could just wrap steps in Number.

Reply:
ok, i reverted